blockbase/inc/wp-customize-palette-control.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Color Palette Customizer Control
+ *
+ * Props to https://github.com/HardeepAsrani/o2 go here
```

Comment:
What's the best way to do this?

Reply:
I'm not sure we need to do we, the code is pretty much a total change